Index: Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Use an official Node.js runtime as a parent image\nFROM node:16\n\n# Set the working directory in the container\nWORKDIR /app\n\n# Copy package.json and package-lock.json to the working directory\nCOPY frontend/react-app/package*.json ./\n\n# Install app dependencies\nRUN npm install\n\n# Copy the rest of your application code to the working directory\nCOPY . .\n\n# Expose a port to communicate with the React app\nEXPOSE 5173\n\n# Start your React app\nCMD [\"npm\", \"run\", \"dev\"]\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Dockerfile b/Dockerfile
--- a/Dockerfile	(revision 5700d5e69bf769ecd90b36006a4fce01f3bd5f88)
+++ b/Dockerfile	(date 1721031616993)
@@ -18,3 +18,23 @@
 
 # Start your React app
 CMD ["npm", "run", "dev"]
+# Use an official Node.js runtime as a parent image
+FROM node:16
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy package.json and package-lock.json to the working directory
+COPY frontend/react-app/package*.json ./
+
+# Install app dependencies
+RUN npm install
+
+# Copy the rest of your application code to the working directory
+COPY . .
+
+# Expose a port to communicate with the React app
+EXPOSE 5173
+
+# Start your React app
+CMD ["npm", "run", "dev"]
