Index: frontend/Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/frontend/Dockerfile b/frontend/Dockerfile
--- a/frontend/Dockerfile	(revision 539b92978163e3b7b7f125c90adc0042592fed58)
+++ b/frontend/Dockerfile	(date 1721031064434)
@@ -1,0 +1,20 @@
+# Use an official Node.js runtime as a parent image
+FROM node:16
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy package.json and package-lock.json to the working directory
+COPY frontend/react-app/package*.json ./
+
+# Install app dependencies
+RUN npm install
+
+# Copy the rest of your application code to the working directory
+COPY . .
+
+# Expose a port to communicate with the React app
+EXPOSE 5173
+
+# Start your React app
+CMD ["npm", "run", "dev"]
\ No newline at end of file
